README.md

```diff
@@ -1,32 +1,36 @@
 # Dennis -- helping to release and ship it
 
-## Install
+## Setup
+
+- It's advised you use a cache directory so Dennis doesn't need to clone the repositories for every release.
+
+```
+mkdir -p ~/.dennis
+```
+
+- You may benefit from either adding a Bash alias, or from creating a one-liner script within your project, for running the releases
```

Comment:
We are in fact no longer going to need any setup at all for Dennis. We simply run the script that will exist in the repo